generators/typescript/sdk/generator/src/test-generator/JestTestGenerator.ts

```diff
@@ -202,6 +202,10 @@ describe("test", () => {
                         code`process.env.${schema.passwordEnvVar ?? "TESTS_PASSWORD"} || "test"`
                     ]);
                 },
+                oauth: () => {
+                    // noop
+                    // TODO(amckinney): support oauth in integration tests
```

Comment:
cc @amckinney  i had to upgrade the ir and just nooping oauth in a couple places